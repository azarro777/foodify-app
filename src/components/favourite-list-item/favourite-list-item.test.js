import React from 'react';
import FavouriteListItem from './favourite-list-item';
import { shallow } from 'enzyme';

describe('all props', () => {
  const props = {
    menuItem: []
  };

  describe('Testing <FavouriteListItem/>', () => {
    it('FavouriteListItem have rendered correctly', () => {
      const favourite = shallow(<FavouriteListItem {...props} />);
      expect(favourite).toMatchSnapshot();
    });
  });
});
