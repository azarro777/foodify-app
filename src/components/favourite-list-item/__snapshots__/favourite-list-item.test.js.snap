// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`all props Testing <FavouriteListItem/> FavouriteListItem have rendered correctly 1`] = `ShallowWrapper {}`;
